utils/create-validate-install-plans.ts

```diff
@@ -1,70 +1,105 @@
-'use strict';
-const path = require('path');
-const { readYamlFile, passedProcessArguments } = require('./helpers');
-const {
+import {
+  InstallPlanDestination,
+  InstallPlanDirectiveInput,
+  InstallPlanMutationVariable,
+  InstallPlanOperatingSystem,
+  InstallPlanTargetInput,
+  InstallPlanTargetType,
+} from './types/InstallPlanMutationVariables';
+import {
```

Comment:
Super nitpick - Personally I think its easier to read `import type {...} from '...'` when it comes to importing types (especially sinbce this file has so many), however I think this is something we can achieve throughout converting to TS.